api/ocm/plugin/ppi/interface.go

```diff
@@ -211,3 +219,28 @@ type Command interface {
 
 	Command() *cobra.Command
 }
+
+type TransferHandler interface {
+	GetName() string
+	GetDescription() string
+	GetQuestions() []DecisionHandler
+}
+
+type DecisionHandler interface {
+	GetQuestion() string
+	GetDescription() string
+	GetLabels() *[]string
+
+	DecideOn(p Plugin, question interface{}) (bool, error)
+}
```

Comment:
Considering these are the interfaces developers are supposed to implement, I think they could use a little more documentation.